doc/api/api_changes/2017-10-24-DS.rst

```diff
@@ -0,0 +1,5 @@
+`StemContainer` now stores `LineCollection`
+-------------------------------------------
+
+`StemContainer` objects now store a `LineCollection` object instead of a list
+of `Line2D` objects for stem lines plotted using `ax.stem`.
```

Comment:
Can you expand on the consequences of this a bit?  I would have to go look up the code to see how to do a migration.

Adding a note about the speed up would help mollify the angry user who's code we just broke ;)

Reply:
I've had a go, but not too sure on what to put - if anyone has a better suggestion feel free to push to my branch.